### ISSUE TITLE
Merged list behavior

### ISSUE THREAD

**Issue by ntderosu-code** [ntderosu-code/PDF-Assist11y#10]
When merging lists in the structure elements panel, they should be done semantically, LI must be a child of L, etc...
